### ISSUE TITLE
[DESIGN] "small_toolbar" branch design issues.

### ISSUE THREAD

**Issue by me4oslav** [fossfreedom/alternative-toolbar#42]
I just installed the small_toolbar branch and noticed the following issues:
  -  The joined buttons should go 5px to the top;
  -  The should start on the same point on the X axis that where the rest of the sidebar elements start;
However before fixing the second issue, this needs fixing: https://github.com/fossfreedom/alternative-toolbar/issues/40

**Comment by fossfreedom** [comment]
?  do you mean add a 5px margin-bottom between the buttons and the playbar?

At the moment it is centered - so you want it left-aligned as RB does at the moment?

**Comment by me4oslav** [comment]
Yes, between the top of the playbar and the bottom of this inline toolbar a 5px whitespace would be good.
And yes, keep it left aligned, so we won't break the HIG. :)

**Comment by fossfreedom** [comment]
ok - I've reverted all of this.  Initially I was playing around with the small toolbar - making it centralised.

Given that it is apparently not "HIG" - there isnt much purpose to keep playing here.

Note - I happened to note that on Google+ Ozon, someone was complaining about the small toolbar appearing like buttons.  This doesnt occur on Adwaita - nor does it occur on Ubuntu Ambiance and Radiance.  Thus, this button issue is probably due to the theme you are using.

I've made a small tweak to force the "linked" Gtk Style on the box holding the buttons.  If this doesnt fix this theme issue that person was observing then suggest raise a bug on whichever theme you were using.
